CONFIGURATION.md

```diff
@@ -6,8 +6,59 @@
 [![Development Phase](https://github.com/spiffe/spiffe/blob/main/.img/maturity/dev.svg)](https://github.com/spiffe/spiffe/blob/main/MATURITY.md#development)
 [![Artifact Hub](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/spiffe)](https://artifacthub.io/packages/search?repo=spiffe)
 
+## Image Configuration
+
+Registries contain container images which are organized by tags.  An organization
+may wish to hold the images in a local repository for better image management
+or to capture local builds of SPIRE when on-site source code management is required.
+
+The values associated with global image controls include:
+
+| Path               | Type   | Default           |
+| ------------------ | ------ | ----------------- |
+| image.registry     | string | ghcr.io           |
+| image.registryPort | int    |                   |
+| image.tag          | string | 1.8.0             |
+
+These controls alter all image defaults, providing a convenient way to ensure
+all SPIRE components use the same set of images. These defauls can be overridden
+by specific component image controls.
+
 ## Agent Configuration
 
+### Agent Image Controls
+
+The agent is controlled by a container defintion which pulls the agent's image
+from a container repository. This image contains the spire-agent executable
+which connects to the spire-server through a procedure known as agent-attestation.
+
+The values associated with the agent image include:
+
+| Path                     | Type   | Defaults             |
+| ------------------------ | ------ | -------------------- |
+| agent.image.name         | string | spire/spire-agent    |
+| agent.iamge.registry     | string | {image.registry}     |
+| agent.iamge.registryPort | int    | {image.registryPort} |
+| agent.iamge.tag          | string | {image.tag}          |
```

Comment:
iamge misspelled here.

Also, would it make sense to mention the default when image.X is not specified after the {image.X} reference so they know what it really is?

Reply:
Thanks for the spelling find.

My first attempt did this.  I don't know if I was too tired to do it well, but I found a few issues in my first attempt to move along your suggestions:

* You need to explain that there are multiple defaults. 
* You then need to explain in which order the defaults apply, and that they might not apply if they aren't set.

That meant I had to explain how to read the defaults and then that a default might exist and what order they applied, if they existed.  This also duplicated a lot of the information in the image.* area.

Then I realized that if I just chain them like I'm doing here, it reuses the explanations of how to use the image.* items and that removed two-thirds of the sentences.

So, it was considered, was the initial attempt, and was discarded.  Maybe I botched the initial approach, but we can always pivot from this at a later date.

Reply:
The problem is, now the user cant look at values, and cant look at docs, so they are left to look at code, and very complicated code currently at that. So, not a great user experience. Even if it was ambiguous if it said "{image.name} | gch.io" or whatever, it would still be much more understandable then the current iteration?